core/domain/state_domain.py

```diff
@@ -1910,29 +1910,27 @@ def update_interaction_hints(self, hints_list):
         self._update_content_ids_in_assets(
             old_content_id_list, new_content_id_list)
 
-    def update_interaction_solution(self, solution_dict):
+    def update_interaction_solution(self, solution):
         """Update the solution of interaction.
 
         Args:
-            solution_dict: dict or None. The dict representation of
-                Solution object.
+            solution: Solution object.
```

Comment:
Still incomplete, it should be `solution: Solution. The solution object.` The item, just after the variable name is its type, which in this case is just 'Solution', and then explain what it is.

Reply:
done.